Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 The Service Manager Authors
```

Comment:
Seems like most Makefiles put .PHONY on targets that do not create files (that are basically commands to be executed) and that should always be ran (even if a file with the target's name is present).

Maybe it would make sense to do the same here

Reply:
https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile